GooglePlayInstant/Editor/PlayInstantBuilder.cs

```diff
@@ -222,7 +222,10 @@ public static void LogError(string message)
             }
         }
 
-        private static bool IsHeadlessMode()
+        /// <summary>
+        /// Returns true if this is a headless command line build, and false if it is an Editor-initiated build.
+        /// </summary>
+        public static bool IsHeadlessMode()
```

Comment:
Is this the right place for this function if it's public? Maybe move it to WindowUtils?

Reply:
Given the documentation I chose "if this is a headless command line build", it's located appropriately, but I'm happy to move it to WindowUtils.

Reply:
That's fair, it doesn't hurt to keep it there for now.

Reply:
Well I've already done the changes, and they are in the latest commit.